includes/compat/subscriptions/class-wc-payment-gateway-wcpay-subscriptions-compat.php

```diff
@@ -84,7 +84,8 @@ private function is_changing_payment_method_for_subscription() {
 	 * @return Payment_Information An object, which describes the payment.
 	 */
 	protected function prepare_payment_information( $order ) {
-		if ( ! wcs_order_contains_subscription( $order->get_id() ) && ! $this->is_changing_payment_method_for_subscription() ) {
+		$is_changing_payment = $this->is_changing_payment_method_for_subscription();
+		if ( ! wcs_order_contains_subscription( $order->get_id() ) && ! $is_changing_payment ) {
```

Comment:
Can we move `! $is_changing_payment` at first place in this condition - to make code a little bit lazier =)

Reply:
Lazy is good! Done.